Sources/TuistSupport/System/System.swift

```diff
@@ -179,6 +182,8 @@ public enum SystemError: FatalError {
 }
 
 public final class System: Systeming {
+    public var verbose: Bool = false
```

Comment:
This might end up being accessed from multiple threads so I'd make it thread-safe. Maybe a good use case for using property wrappers? https://www.vadimbulavin.com/swift-atomic-properties-with-property-wrappers/

Reply:
Typically these variables are configured on the main thread, I can't imagine we would ever be setting this more than once?

Reply:
> Typically these variables are configured on the main thread

That's an assumption about how we think things will be used in the future, but nothing prevents future developers using them differently. As the codebase grows, assumptions might turn into future issues so I'd avoid it. Especially in this case where we are dealing with a global instance with a global state.